### PR TITLE
Add a test for cedar-policy/cedar#1074

### PR DESCRIPTION
**Pull request by shaobo-he-aws**
*Issue #, if available:*

*Description of changes:*

### CODE REVIEW

**Review by cdisselkoen — APPROVED** [review]
Looks good, once CI is passing

**Review by khieta — APPROVED** [review]
Lgtm, but don't merge until after the changes to `cedar` are merged and CI is passing.

**Pull request merged**
